.github/workflows/sdks.yml

```diff
@@ -97,18 +99,27 @@ jobs:
             TAG="${{ needs.get-latest-toolchain.outputs.trunk-version }}"
           fi
           echo "tag=$TAG" >> $GITHUB_OUTPUT
-          KEY="$TAG-${{ matrix.arch }}-ndk-27b-c-ares-sdk"
-          echo "key=$KEY" >> $GITHUB_OUTPUT
+          KEY="$TAG-${{ matrix.arch }}-ndk-${NDK_VERSION}-arch-sdk"
+          echo "arch-key=$KEY" >> $GITHUB_OUTPUT
+          echo "key=$TAG-ndk-${NDK_VERSION}-bundle" >> $GITHUB_OUTPUT
           if ${{ matrix.arch == 'x86_64' }}; then
             echo "aarch64-key=${KEY/x86_64/aarch64}" >> $GITHUB_OUTPUT
             echo "arm-key=${KEY/x86_64/armv7}" >> $GITHUB_OUTPUT
           fi
-      - name: Get cached SDK
+      - name: Get cached SDK bundle
+        id: cache-bundle
+        uses: actions/cache@v4
+        if: ${{ matrix.version != 'trunk' }}
```

Comment:
I'm seeing strange errors when trying to build packages with the new trunk 6.1 SDK bundle, so I'm disabling using it on the CI until that's fixed.